samples/nrf9160/lwm2m_client/src/main.c

```diff
@@ -544,6 +564,10 @@ void main(void)
 	}
 #endif
 
+#if defined(CONFIG_LWM2M_CLIENT_UTILS_SIGNAL_MEAS_INFO_OBJ_SUPPORT)
+	k_timer_start(&ncell_meas_work_timer, K_SECONDS(1), K_SECONDS(60));
```

Comment:
Would it make sense to have these as Kconfig parameters?

Reply:
Maybe the interval would be.